airbyte-integrations/connectors/source-freshdesk/integration_tests/catalog.json

```diff
@@ -0,0 +1,129 @@
+{
+  "streams": [
+    {
+      "name": "agents",
```

Comment:
Do we need this file? Haven't seen it used anywhere

Reply:
Looks like someone someone by mistake added it here instead of at secrets